nncf/quantization/adjust_padding.py

```diff
@@ -0,0 +1,109 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from collections import namedtuple
+from typing import NamedTuple
+
+import networkx as nx
+import torch
+from torch import nn
+
+from nncf.dynamic_graph.context import Scope
+from nncf.dynamic_graph.graph import NNCFGraph
+from nncf.layers import NNCFConv2d
+from nncf.module_operations import UpdatePaddingValue
+from nncf.nncf_network import NNCFNetwork
+from nncf.quantization.layers import BaseQuantizer
+from nncf.quantization.layers import QuantizationMode
+from nncf.quantization.layers import QuantizerConfig
+from nncf.quantization.layers import SymmetricQuantizer
+
+
+class AdjustPaddingArgs(NamedTuple):
+    weight_bitwidth: int
+    activation_quantizer: BaseQuantizer
+    quantized_module: nn.Module
+    module_scope: Scope
+
+
+class AdjustPadding:
```

Comment:
Since it does not actually adjust padding values, should probably rename it to `CalculatePaddingAdjustment`.